docs/source/install/__packages_beta_warning.rst

```diff
@@ -0,0 +1,7 @@
+.. warning::
+
+   Packages based installation is currently is beta and upgrades are not yet
```

Comment:
I also think we should simply remove AIO and Chef / Ansible / Puppet installation documents.